Snapshot terminology update (#850)
chromaui/chromatic-docs: src/content/account/billing.mdx

Reviewer: ```suggestion
Invoices are generated each month on your account's Billing page. Click to see a PDF of the amount owed and a breakdown of billed snapshot usage.
```

Reviewer: "snapshot" -> "billed snapshot"

## Snapshots

The monthly bill is calculated based on the number of billed [snapshots](/docs/snapshots). There are two types of snapshots:

> - **captured (1 billed snapshot):** Chromatic performs the full suite of work, including taking screenshots or capturing accessibility data, and running diffs.

Reviewer 2: ```suggestion
- **Captured (1 billed snapshot):** Chromatic performs the full suite of work, including taking screenshots or capturing accessibility data, and running diffs.
```

> - **turbosnap (0.2 billed snapshot):** Chromatic identifies tests that have no code changes associated with them or their dependencies and copies over the snapshots from existing baselines for them.

Reviewer 2: ```suggestion
- **Turbosnap (0.2 billed snapshot):** Chromatic identifies tests that have no code changes associated with them or their dependencies and copies over the existing snapshots from previous baselines for them.
```

When at the beginning of a sentence we should capitalize turbosnap (noun for turbo snapshot) even if there's confusion with the proper noun for the feature TurboSnap. That said, I'd be OK with always styling it as `TurboSnap[s]` instead of having to remember which spelling is correct.

## How we count billed snapshots

Each test can lead to multiple snapshots depending on the number of browsers/devices, modes, and test types you run. [Visual](/docs/visual) and [accessibility tests](/docs/accessibility) incur billed snapshots; [interaction tests](/docs/interactions) don't.

Billed snapshots are calculated based on the number of tests, builds, browsers, and modes. You can use our [snapshot calculator](https://www.chromatic.com/snapshot-calculator) to estimate your monthly snapshot usage and see the underlying math.

```
visual snapshots = Tests x Builds x Browsers x Modes
accessibility snapshots = Tests x Builds x Modes
```

### Without TurboSnap enabled

> Chromatic captures a new snapshot for every test in every build. Therefore, the total billed snapshots are calculated as follows:

Reviewer 2: ```suggestion
Chromatic captures a new snapshot for every test in every build resulting in higher usage. The total billed snapshots are calculated as follows:
```

```
billed snapshots = visual snapshots + accessibility snapshots
```

### With TurboSnap enabled

> Chromatic copies over snapshots from existing baselines for turbosnaps. For the remaining stories, Chromatic captures new snapshots as usual. Therefore, the total billed snapshots are calculated as follows:

Reviewer 2: ```suggestion
Chromatic copies over snapshots from existing baselines. We refer to these as "turbosnaps", which cost 0.2 billed snapshots each. For the remaining stories, Chromatic captures new snapshots as usual. Therefore, the total billed snapshots are calculated as follows:
```

```
Visual snapshots = Tests with changes x Builds x Browsers x Modes
Accessibility snapshots = Tests with changes x Builds x Modes

turbosnaps = (Tests - Tests with changes) x Builds x Browsers x Modes
billed snapshots from turbosnaps = turbosnaps * 0.2

billed snapshots = visual snapshots + accessibility snapshots + billed snapshots from turbosnaps
```

### Billed snapshots are counted at the account level

> If your account has multiple projects, we sum the billed snapshots each project incurs to get your total usage. Chromatic doesn't break down billing per project.

Reviewer: Is this true? We do show usage on the project manage screen and break the raw snapshot usage down per-project on the usage report

Reviewer 2: It's technically true but not really true in spirit. We should change the language.

A couple more things to note:

- Your [billing page](/docs/billing#view-current-months-usage) will include a breakdown of captured snapshots vs turbosnaps.
- If you use our free [Publish Storybook](/docs/publish) service only (i.e., UI Test and UI Review is disabled) you won't incur billed snapshots.

### View current month's usage

Find the current month's usage in the Plan section of the Billing page.

![monthly billed snapshots used](../../images/article-view-snapshots-billing-screen.png)

### Additional billed snapshots

Each plan comes with a specific number of included billed snapshots. If your usage reaches that amount:

1. **Free plan:** Review and testing will be paused once you use all 5,000 included billed snapshots per month. You can resume testing by upgrading to a paid plan or waiting until your included billed snapshots reset at the end of the billing cycle. ![Builds blocked because the account hit the monthly snapshot limit](../../images/build-blocked.png)

> 2. **Paid plans:** Review and testing will not be interrupted. Any billed snapshots beyond the included amount are charged at the end of the month.

Reviewer 2: ```suggestion
2. **Paid plans:** Review and testing will not be interrupted. Additional billed snapshots beyond the included amount are charged at the end of the billing period.
```

> For information on the number of billed snapshots included in each plan and the cost of additional snapshots, please refer to the [pricing page](https://www.chromatic.com/pricing). You can also set [usage alerts](/docs/billing#usage-alerts) to notify a designated email address when your usage reaches a certain number of billed snapshots.

Reviewer 2: ```suggestion
For information on the number of billed snapshots included in each plan and the cost of additional snapshots, refer to the [pricing page](https://www.chromatic.com/pricing). You can also set [usage alerts](/docs/billing#usage-alerts) to notify a designated email address when your usage reaches a certain number of billed snapshots.
```

> The **usage section** provides a per-project summary of Snapshots and TurboSnaps utilized (only available for billing periods starting in August 2024).

Reviewer: ```suggestion
The **usage section** provides a per-project summary of Visual, Accessibility and TurboSnap snapshots utilized.
```

Reviewer: Also we can probably drop the August 2024?

> The **invoice** includes details of the active plan for the billing period, the base number of billed snapshots included, charges for additional snapshots, and a link to download a PDF copy of the invoice (available for all billing periods).

Reviewer: ```suggestion
The **invoice** includes details of the active plan for the billing period, the base number of billed snapshots included, charges for additional billed snapshots, and a link to download a PDF copy of the invoice (available for all billing periods).
```

> ![Example of a usage report that shows a breakdown of billed snapshots by project and the invoice details](../../images/usage-report.png)

Reviewer: I think this is out of date, it says "Visual" not "Snapshots" now. I think generally avoiding "Snapshots" by itself is a good idea :)
